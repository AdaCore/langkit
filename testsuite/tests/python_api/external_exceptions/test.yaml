driver: lkt_build_and_run

config:
  library:
    external_exceptions:
      - exception: Helpers.My_External_Exception
        doc_section: libfoolang.helpers

  extra_docs:
    libfoolang.helpers.my_external_exception: |
      Documentation of ``My_External_Exception``.

py_script: main.py
