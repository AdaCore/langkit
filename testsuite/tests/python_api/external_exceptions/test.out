main.py: Running...
Calling FooNode.p_prop...
  Got an exception:
    MyExternalException: my exception message
main.py: Done.
Done
